### ISSUE TITLE
Question about Faster RCNN pretrained model

### ISSUE THREAD

**Issue by Noba1anc3** [dragen1860/TensorFlow2.0Tutorials#55]
Hello,
I am wondering if the pretrained model of Faster RCNN is trained on ImageNet or COCO, and how about the performance?
Thanks a lot.

**Comment by BIGBIGBrotherAI** [comment]
Sorry,I have no idea.



---Original---

**Comment by tensor2man** [comment]
According to my points and experiences, the performance of the pre-trained model is not good.
In order to acquire the accuracy you want, you have to train the pre-trained model more.
In my case, I used to use the pre-trained model to initialize the weight parameters of that network at the beginning.
